### ISSUE TITLE
[1.16] Aquamirae mod - Enchantment crashes the server

### ISSUE THREAD

**Issue by XionioXMaster** [Loli-Server/LoliServer#329]
**Minecraft Version(我的世界版本) :** 1 16 5
**LoliServer buildVersion(构建版本) :** Provide LoliServer  buildVersion: "[LoliServer-1.16.5-******-server.jar](https://github.com/Loli-Server/LoliServer/actions/runs/1782661183)"

**Operating System(操作系统) :** unbuntu 20

**Concerned mod / plugin(有关的插件/Mod) :** [Provide mod / plugin name and download link if possible](https://www.curseforge.com/minecraft/mc-mods/ob-aquamirae/files/3549955)

**Logs :** [Link(s) of logs and crash reports if you have, (https://paste.loliidc.cn)](https://paste.loliidc.cn/?cd8781443500e449#Aq5FaDmqr9PbgBpPdFJf4xEqu9E1HWTZG1Zo1yTrWfsb)

**Steps to Reproduce(复现步骤) :**
 1. Crash Cause: Crystallize enchantment
 2. This enchantment delays your death by 40 seconds.
In short, you die and immediately resurrect
And all the things you have
And if you wait for those 40 seconds
That crash
